js/main.js

```diff
@@ -1,49 +1,172 @@
+      + '</div>'
+      + '</div>';
+    }
+  }
+}
+createProjectsSeciton();
+
+// Project popups functionality
+const projectDiv = document.createElement('div');
+const projectBttns = document.querySelectorAll('.popup');
+
+function addEventListenerList() {
+  for (let i = 0; i < projectBttns.length; i += 1) {
+    projectBttns[i].addEventListener('click', (event) => {
+      const projectID = event.target.id;
+      grayBgDiv.id = 'overlay-popup-projects';
+      grayBgDiv.classList.add('overlay');
+      projectDiv.id = 'overlay-popup';
+      projectDiv.classList.add('overlay');
+
+      // Techs Creation
+      let techsHTMLDetail = '';
+      for (let j = 0; j < projects[i].techs.length; j += 1) {
+        techsHTMLDetail += `<li class="popup-li popup-items-basic">${projects[i].techs[j]}</li>`;
+      }
+
+      projectDiv.innerHTML = `<div class="image-container-popup"><img  class="popup-project-image" src="${projects[projectID].imgUrl}">`
+        + '<img id="cancel-detail" src="./images/buttons/cancel-detail-2.png"></div>'
+        + '<div class="popup-content">'
+        + `<h2 class="popup-title font-family font-color">${projects[projectID].name}</h2>`
+        + `<ul class="popup-text font-family">${
+          techsHTMLDetail
+        }</ul>`
+        + `<p class="font-family popup-text">${projects[projectID].description} Lorem Ipsum is simply dummy text of the printing`
+        + 'and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown'
+        + 'printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and'
+        + 'typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer'
+        + 'took a galley of type and scrambled it 1960s with the relea</p>'
+        + '<div class="bttn-popup-container"><a href="#" class="bttn-popup font-family">See Source</a>'
+        + '<a href="#" class="bttn-popup font-family">See Live</a></div>'
+        + '</div>';
+      grayBgDiv.appendChild(projectDiv);
+      document.body.appendChild(grayBgDiv);
+
+      const cancelDetail = document.querySelector('#cancel-detail');
+      cancelDetail.addEventListener('click', () => {
+        projectDiv.remove();
+        grayBgDiv.remove();
+      });
+    });
+  }
+}
+addEventListenerList();
+
+/*
+
+<div class="grid-item-1 project-image-container"></div>
+<div class="grid-item-2">
+  <div class="flex-container flex-column-center-axis works-title font-family">
+    <div class="flex-item-1 flex-container flex-column-center-axis">
+      <h3 class="card-font card-Title font-family"></h3>
+    </div>
+    <div class="flex-item-2 flex-container techs-items-font font-family">
+    </div>
+    <div class="flex-item-1 flex-container flex-column-center-axis">
+      <a href="#" class="bttn font-family popup" id="0">See Project</a>
+    </div>
+  </div>
+</div>
+
+*/
```

Comment:
- [ ] I am inviting you to remove all the commented code from the `index.js` file as it would be recommended to not push the commented code as it makes the codebase little messy and unclear.

Reply:
Thanks for the headups! I completely forgot to remote it. Fix was implemented :wrench: